__tests__/registries/npm-registry.js

```diff
@@ -6,6 +6,11 @@ import NpmRegistry from '../../src/registries/npm-registry.js';
 import {BufferReporter} from '../../src/reporters/index.js';
 import homeDir, {home} from '../../src/util/user-home-dir.js';
 
+function basicAuth(username, password): string {
+  const pw = new Buffer(String(password), 'base64').toString();
```

Comment:
`Buffer.from` should be used on these two lines instead of the `Buffer` constructor (which has been deprecated).

Reply:
Updated, should I squash the commits?

Reply:
Great, thanks!

You can squash this if you want, but it doesn't matter, We typically use squash merging, so Github will ensure it's a single commit either way.